server/consumables.lua

```diff
@@ -1,64 +1,37 @@
 local QBCore = exports['qb-core']:GetCoreObject()
 
 ----------- / alcohol
-
-QBCore.Functions.CreateUseableItem("vodka", function(source, item)
-    TriggerClientEvent("consumables:client:DrinkAlcohol", source, item.name)
-end)
-
-QBCore.Functions.CreateUseableItem("beer", function(source, item)
-    TriggerClientEvent("consumables:client:DrinkAlcohol", source, item.name)
-end)
-
-QBCore.Functions.CreateUseableItem("whiskey", function(source, item)
-    TriggerClientEvent("consumables:client:DrinkAlcohol", source, item.name)
-end)
-
------------ / Eat
-
-QBCore.Functions.CreateUseableItem("sandwich", function(source, item)
-    local Player = QBCore.Functions.GetPlayer(source)
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Eat", source, item.name)
-end)
-
-QBCore.Functions.CreateUseableItem("twerks_candy", function(source, item)
-    local Player = QBCore.Functions.GetPlayer(source)
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Eat", source, item.name)
-end)
-
-QBCore.Functions.CreateUseableItem("snikkel_candy", function(source, item)
-    local Player = QBCore.Functions.GetPlayer(source)
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Eat", source, item.name)
-end)
-
-QBCore.Functions.CreateUseableItem("tosti", function(source, item)
-    local Player = QBCore.Functions.GetPlayer(source)
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Eat", source, item.name)
-end)
-
------------ / Drink
-
-QBCore.Functions.CreateUseableItem("water_bottle", function(source, item)
-    local Player = QBCore.Functions.GetPlayer(source)
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Drink", source, item.name)
-end)
-
-QBCore.Functions.CreateUseableItem("coffee", function(source, item)
-    local Player = QBCore.Functions.GetPlayer(source)
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Drink", source, item.name)
-end)
-
-QBCore.Functions.CreateUseableItem("kurkakola", function(source, item)
-    local Player = QBCore.Functions.GetPlayer(source)
-	if not Player.Functions.RemoveItem(item.name, 1, item.slot) then return end
-    TriggerClientEvent("consumables:client:Drink", source, item.name)
-end)
+for cAl in pairs(ConsumablesDrink) do
```

Comment:
```suggestion
for cAl in pairs(ConsumablesAlcohol) do
```

Reply:
Just spotted that one mate, already handled